website/frontend/src/components/blocks/Header/children/Menu/Menu.module.scss

```diff
@@ -18,23 +18,63 @@
   }
 }
 
-.link {
+%menuItem {
+   color: inherit;
+   height: 100%;
+   padding: 28px 10px;
+   transition: all 0.25s;
+   
+   @include sm {
+      padding: 5px;
+   }
+   
+   &:hover {
+      color: $textColorSecondary;
+   }
+}
+
+.link { 
   display: block;
-  padding: 28px 10px;
   text-align: center;
-  color: inherit;
-  transition: all 0.25s;
   cursor: pointer;
-
-  @include sm {
-    padding: 5px;
-  }
-
-  &:hover {
-    color: $textColorSecondary;
-  }
+   
+  @extend %menuItem;
 
   &.active {
     box-shadow: inset 0 -3px 0 $successColor;
   }
 }
+
+.sidebarBtn {
+   display: flex;
+   flex-flow: column nowrap;
+   align-items: flex-end;
+   padding: 18px 10px;
+   
+   @extend %menuItem;
+   
+   @include sm {
+      align-items: center;
+   }
+
+   &.loading {
+      justify-content: center;
+
+     &::before {  
+       content: 'Loading...';
+     }
+   }
+
+   & > span {
+      text-align: left;
+      overflow-wrap: anywhere;
+
+      &.small {
+        font-size: $fontSizeSmall;
+      } 
+
+      &:not(:last-child) {
+         margin-bottom: 2px;
+      }
+   }
```

Comment:
`overflow-wrap: anywhere` has this behaviour, I think we should drop it:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/67055679/165166544-448e315b-1929-4473-b2da-3a97fbcdf309.png">
`text-align: right` seems to be more appropriate here and you can specify it in a parent